apps/a11y-tests/scripts/generate-sarif-reports.js

```diff
@@ -0,0 +1,27 @@
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const { getSarifReport } = require('../lib/getSarifReport');
+
+const components = [
+  {
+    name: 'Button',
+    url: '#/examples/button'
+  },
+  {
+    name: 'TextField',
+    url: '#/examples/textfield'
+  }
+];
+
+components.forEach(async component => {
+  try {
+    const report = await getSarifReport(component.url);
+    mkdirp.sync(path.join(__dirname, '../dist/reports/'));
+    fs.writeFileSync(path.join(__dirname, `../dist/reports/${component.name}.sarif`), JSON.stringify(report), { encoding: 'utf8' });
```

Comment:
Should probably use `path.resolve` here instead of `path.join`

Reply:
Fixed.